python/ray/experimental/state/api.py

```diff
@@ -1140,11 +1141,13 @@ def get_log(
         filename: Name of the file (relative to the ray log directory) to be retrieved.
         actor_id: Id of the actor if getting logs from an actor.
         task_id: Id of the task if getting logs generated by a task.
-        pid: PID of the worker if getting logs generated by a worker.
+        pid: PID of the worker if getting logs generated by a worker. When querying
+            with pid, one of node_id or node_ip has to supplied.
```

Comment:
May be say, 
When querying with pid, either node_id or node_ip must be supplied.